### PR TITLE
Set up Node in TravisCI

### LINKED ISSUE #1: Set up Node in TravisCI
## TODO

- [x] Enable [Node.js v10 on TravisCI](https://docs.travis-ci.com/user/languages/javascript-with-nodejs/)

- [x] Install npm dependencies using `npm ci`

### PR DESCRIPTION
**Pull request by hariclerry**
### What does this PR do?
This PR creates a .travis.yml file and adds configurations for Nodejs.

### What are the changes made?
- Created .travis.yml file
- Added Node configuration to the .travis.yml file
- Ran npm ci

### Screen shots
None

### CODE REVIEW

**Comment by hariclerry** [comment]
Hey all, I have a few questions regarding this PR;
1. Should I add the repository to my Travis CI account to ensure the setup is working or are we doing this later?
2. Should I add other commands that may be necessary now or later, for example, the `service` command? please advise accordingly. 
3. Please advise on the git workflow, for example, branch naming, commit messages etc.

**Comment by sovoid** [comment]
@hariclerry the service commands should follow based on the issue. This issue explicitly says set up nodejs so that's what should be done in the PR. Feel free to add the repo to your Travis I account since we would need to run the code there

**Comment by hariclerry** [comment]
Noted @und3fined-v01d, I have added the repo to my Travis CI account.

**Comment by sovoid** [comment]
@hariclerry please install the TravisCI app on this repo as well so that we can see the logs as well

File: .travis.yml [hunk 1]
```diff
@@ -1,77 +1,7 @@
...
-  irc:
-    channels:
-      - "chat.freenode.net#wikimedia-dev"
-    template:
-      - "%{repository}#%{build_number} (%{branch} - %{commit} %{author}): %{message} - %{build_url}"
-    on_success: change
-    on_failure: always
+script:  
+  - npm ci
+
```
Comment (sovoid) [inline .travis.yml]: Please remove the extra newline

**Review by sovoid — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (hariclerry) [reply .travis.yml]: This has been fixed.

**Pull request merged**